src/integrationTest/java/com/mongodb/hibernate/query/select/temporal/SelectionInstantIntegrationTest.java

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright 2025-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.hibernate.query.select.temporal;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.time.Instant;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Stream;
+import org.hibernate.testing.orm.junit.DomainModel;
+import org.junit.jupiter.params.provider.Arguments;
+
+@DomainModel(annotatedClasses = {SelectionInstantIntegrationTest.Item.class})
+class SelectionInstantIntegrationTest
+        extends AbstractSelectionTemporalIntegrationTest<SelectionInstantIntegrationTest.Item, Instant> {
+
+    private static final List<Item> ITEMS = List.of(
+            new Item(1, Instant.parse("2025-01-04T10:05:01Z")),
+            new Item(2, Instant.parse("2025-05-04T14:30:15Z")),
+            new Item(3, Instant.parse("2025-12-04T23:59:59Z")));
+
+    private static List<Item> getTestItems(int... ids) {
+        return Arrays.stream(ids)
+                .mapToObj(id -> ITEMS.stream()
+                        .filter(c -> c.id == id)
+                        .findFirst()
```

Comment:
Not that it matters much, but `findAny` is conceptually the right method here.


```suggestion
                        .findAny()
```